main.go

```diff
@@ -242,6 +227,14 @@ func runCLIApp(c *cli.Context) (err error) {
 	logger.Infof("GCSFuse mount command flags: %s", util.Stringify(*flags))
 	logger.Infof("GCSFuse mount config flags: %s", util.Stringify(*mountConfig))
 
+	if flags.StatCacheCapacity != DefaultStatCacheCapacity {
```

Comment:
[no action required] this will not cover the case when user explicitly pass default value.

Reply:
true. added a comment to reflect the same.